src/inmanta/data/__init__.py

```diff
@@ -198,6 +205,23 @@ def get_order(self) -> PagingOrder:
         """ The order string representing the direction the results should be sorted by"""
         return self.order
 
+    def is_nullable_column(self) -> bool:
+        """Is the current order by column type nullable (optional) or not"""
+        column_type = self.get_order_by_column_type()
+        return typing_inspect.is_optional_type(column_type)
+
+    def coalesce_to_min(self, value_reference: str) -> ColumnNameStr:
```

Comment:
Why no use `ORDER BY ... NULLS  FIRST` ?

Reply:
I use that for ordering, but for comparisons, that are needed for paging, the value has to be an actual value. Comparison with `NULL` always yields `NULL`.

Reply:
I would document this.